backend/dataall/cdkproxy/stacks/environment.py

```diff
@@ -325,13 +325,10 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
             string_value=lf_default_settings_custom_resource.function_name,
             parameter_name=f'/dataall/{self._environment.environmentUri}/cfn/lf/defaultsettings/lambda/name',
         )
-
-        # Glue database custom resource
-        # This Lambda is triggered with the creation of each dataset, it is not executed when the environment is created
+        # Glue database custom resource - Old, to be deleted in future release
```

Comment:
I didn't catch this. Why do we consider this `no_delete` as old? In the main branch, we do some actions on `on_delete`

Reply:
As I understand that is addressed in the PR notes:
```
Deleting the Glue database does not delete the S3 data, but it deletes all Lake Formation permissions, which is critical for production use-cases. We cannot introduce this risk, so we have updated the old custom resource Lambda to not delete anything on the on_delete event and we will leave the deprecated custom resource until the next minor release where we will safely remove it from the dataset stack.
```

Reply:
> In V1.6.0 we will ensure to notify that upgrading to V1.5.0 is a requirement before upgrading to V1.6.0.

I wonder though if there is a way to avoid this

Reply:
Exactly, but as I explain below we might choose to have `no_delete` as the general behavior because deleting the database is a quite dangerous action that customers should be fully aware of